t/alignment/longtable.xml

```diff
@@ -7,7 +7,12 @@
 <document xmlns="http://dlmf.nist.gov/LaTeXML">
   <resource src="LaTeXML.css" type="text/css"/>
   <resource src="ltx-article.css" type="text/css"/>
-  <table frefnum="Table 1" refnum="1" xml:id="S0.T1">
+  <table xml:id="S0.T1">
+    <tags>
+      <tag>Table 1</tag>
```

Comment:
just as a readability remark, I'm unsure what the tag with no attribute signifies. Looking at the diff I assume it is the `frefnum` that was originally on the table element? If so, maybe it warrants a role attribute specifying that?

Reply:
see reply to previous comment about what the lack of `@role` means. Either can be default or explicit, if there's a meaningful name for it.